### PR TITLE
22463_update_collectingEvent_open_api_specs_file

### PR DESCRIPTION
**Pull request by luusteve**
https://redmine.biodiversity.agr.gc.ca/issues/22463

### CODE REVIEW

File: schema/collectingEvent.yml [hunk 1]
```diff
@@ -182,24 +182,69 @@ components:
...
-            verbatimCoordinates:
+            verbatimCoordinateSystem:
               type: string
-              description: verbatim original spatial coordinates of the Location, https://dwc.tdwg.org/terms/#dwc:verbatimCoordinates              
-            coordinateUncertaintyInMeters:
+              description: The spatial coordinate system for the verbatimLatitude and verbatimLongitude or the verbatimCoordinates of the Location. https://dwc.tdwg.org/list/#dwc_verbatimCoordinateSystem
+            verbatimSRS:
+              type: string
+              description: spatial reference system (SRS) upon which coordinates given in verbatimLatitude and verbatimLongitude, or verbatimCoordinates are based. https://dwc.tdwg.org/terms/#dwc:verbatimSRS
+            verbatimElevation:
```
Comment (cgendreau) [inline schema/collectingEvent.yml:191]: can't be a number, verbatim needs to be string

**Review by cgendreau — CHANGES_REQUESTED** [review]
geoReferenceAssertions is missing

File: schema/collectingEvent.yml (hunk 1 above)
Comment (luusteve) [reply schema/collectingEvent.yml:191]: I think you highlighted the wrong part, we're you referring to verbatimElevation and verbatimDepth?

Comment (cgendreau) [reply schema/collectingEvent.yml:191]: all verbatims

**Pull request merged**